RUN_DEPLOY.md

```diff
@@ -0,0 +1,240 @@
+<h1>Document run and deploy app react native</h1>
+
+**This guide builds for Mac OS, does not support Window OS**
+_The running and deploying instructions below are written based on the PIT PORT project, not the sample project._
+
+<h2>Table of Contents</h2>
+
+- [Setup Environment](#setup-environment)
+- [Getting Started](#getting-started)
+- [Getting Started Android](#getting-started-android)
+- [Getting Started iOS](#getting-started-ios)
+- [Project's structures](#projects-structures)
+  - [Introduction](#introduction)
+  - [Main folders, files](#main-folders-files)
+- [Deploy](#deploy)
+  - [Android](#android)
+  - [iOS](#ios)
+
+## Setup Environment
+
+Install the environment (Android, IOS) by following the link: [Setup environment](https://reactnative.dev/docs/environment-setup)
+
+## Getting Started
+
+**The application has 3 environments:**
+
+- **Development**: develop and fix bugs
+  Env file: `.env.development`
+
+- **Staging**: environment to test again before pushing to production
+  Env file: `.env.staging`
+
+- **Production**: environment users can use
```

Comment:
**Production**: `.env.production`
- Objectives:
  - Final version for business.
  - Any bugs will be **hot-fix** in this environment.
- Audiences:
  - Developers (optional)
  - Testers
  - End-users
  - Clients